mason and blacksmiths

Dorodomki opened the pull request:
## About The Pull Request

gives new combat music to the masons and blacksmiths, gives dwarfvish lenguage to the maker's guild artisans except the apprentices and the elfs since no elf must speak dwarfvish and if they do is a grudge to solve

removes the dwarfvish lenguage from the maker's elfs with spritercode that WORKS

## Why It's Good For The Game

cool enough to add 

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

Dorodomki committed:
mason and blacksmith

gives new music to the masons and blacksmiths, gives dwarfvish lenguage to the maker's guild artisans except the apprentices and the elfs

Dorodomki committed:
Merge branch 'main' into post-kaizoku-fixes

ThePainkiller commented on code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm:
```diff
@@ -221,3 +223,7 @@
 	H.change_stat("strength", 1)
 	H.change_stat("endurance", 2)
 	H.change_stat("speed", -1)
+	H.grant_language(/datum/language/dwarvish)
```
This needs a check first to see if the person already has the dwarvish language, to not add the language twice and avoid conflicts.
Also I did tell you before to add it as a chance, not a guarantee.

Dorodomki replied:
![Captura de pantalla (2612)](https://github.com/user-attachments/assets/8e932b72-4472-44cc-be31-df120fbe575d)

just localtested it, the dwarfs don't get duplicated lenguages just one

ThePainkiller replied:
Fix the code still.

Dorodomki committed:
big codefix for the game

it compiled without errors so all cool

Dorodomki committed:
Merge branch 'post-kaizoku-fixes' of https://github.com/Dorodomki/OpenKeep into post-kaizoku-fixes

ThePainkiller commented:
You forgot to add it to legendary smith like you said you would lol

ThePainkiller committed:
Merge branch 'main' into post-kaizoku-fixes

Dorodomki commented:
> You forgot to add it to legendary smith like you said you would lol

let them rot

ThePainkiller commented:
Give language to the legendary classes as well, stop being lazy and commit fully to a change, don't do two PR branches for such a simple change when it's copypasting a code that I literally gave you to paste over.

Dorodomki committed:
legendary smith update

adds the lenguage of dwarfvish to legendary smiths, more attempts to take the lenguage away from tieflings and drows is needed since they aren't civilized races either
